Formula/graphviz.rb

```diff
@@ -48,6 +48,10 @@ class Graphviz < Formula
   end
 
   def install
+    # The following setting makes to prohibit using Homebrew's Python. See :
+    # https://github.com/Homebrew/homebrew-core/issues/19110
+    ENV["PYTHON"]="/usr/bin/python"
```

Comment:
The `python` flexibility is an intentional thing here, or the dependency wouldn't be `depends_on :python`.